### PR TITLE
Fix snapshot

### PR DESCRIPTION
**Pull request by chenshoo**

### CODE REVIEW

File: src/polaris-gql-module-options/polaris-gql-module-options.service.ts [hunk 1]
```diff
@@ -51,11 +52,12 @@ export class GqlOptionsFactoryService implements GqlOptionsFactory {
       subscriptions,
       introspection,
       formatError: polarisFormatError,
-      transformSchema: (schema: GraphQLSchema) => {
+      transformSchema: (schema: any) => {
```
Comment (osher-sade) [inline src/polaris-gql-module-options/polaris-gql-module-options.service.ts:55]: why did you change the type to any?

File: src/routes/routes.service.ts [hunk 2]
```diff
@@ -33,8 +32,8 @@ export class RoutesService {
     const realityId: number = realityHeader ? +realityHeader : 0;
     const snapshotRepository = getConnectionForReality(
       realityId,
-      this.config.getPolarisServerConfig().supportedRealities,
-      getPolarisConnectionManager()
+      this.config.getPolarisServerConfig().supportedRealities as any,
```
Comment (osher-sade) [inline src/routes/routes.service.ts:35]: why did you add `as any`?

File: test/test-server/polaris-server-options-factory/polaris-server-options-factory-service.ts [hunk 3]
```diff
@@ -26,6 +27,15 @@ export const createOptions: () => PolarisServerOptions = () => {
...
     ),
+    connectionManager: {
+      connections: [getPolarisConnectionManager().get()],
+      has: (name: string) => {
+        return name === "default";
+      },
+      get: (name: string) => {
+        return getPolarisConnectionManager().get(name);
+      },
+    } as any, // getPolarisConnectionManager(),
```
Comment (osher-sade) [inline test/test-server/polaris-server-options-factory/polaris-server-options-factory-service.ts:38]: delete the comment

**Review by DoctorVoid — APPROVED** [review]
